### ISSUE TITLE
Can't listen to archives - exception thrown

### ISSUE THREAD

**Issue by github-ronk** [dexterouslogic/plugin.audio.wfmu#1]
Hi

Thanks for the plugin :)

Listening to the live stream works great.

Browsing the archives works.

However, when trying to listen to an episode, the following exception is thrown, and playback does not start.

```
10:38:10 T:140011293869824  NOTICE: Thread LanguageInvoker start, auto delete: false
10:38:10 T:140011293869824  NOTICE: -->Python Interpreter Initialized<--
10:38:10 T:140011293869824  NOTICE: [xbmcswift2] Request for "/archive/feed/KK/" matches rule for function "route_archivefeed"
10:38:11 T:140011293869824   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.AttributeError'>
                                            Error Contents: 'NoneType' object has no attribute 'attrib'
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.audio.wfmu/addon.py", line 68, in <module>
                                                plugin.run()
                                              File "/storage/.kodi/addons/script.module.xbmcswift2/lib/xbmcswift2/plugin.py", line 332, in run
                                                items = self._dispatch(self.request.path)
                                              File "/storage/.kodi/addons/script.module.xbmcswift2/lib/xbmcswift2/plugin.py", line 306, in _dispatch
                                                listitems = view_func(**items)
                                              File "/storage/.kodi/addons/plugin.audio.wfmu/addon.py", line 40, in route_archivefeed
                                                items = _route_archivefeed(show_id)
                                              File "/storage/.kodi/addons/script.module.xbmcswift2/lib/xbmcswift2/xbmcmixin.py", line 81, in wrapper
                                                result = function(*args, **kwargs)
                                              File "/storage/.kodi/addons/plugin.audio.wfmu/addon.py", line 34, in _route_archivefeed
                                                return list(router.route(show_id))
                                              File "/storage/.kodi/addons/plugin.audio.wfmu/resources/router/archivefeed.py", line 28, in _get_list_items
                                                ts = i.find(_get_tag('startDate')).attrib.get('unixTimestamp')
                                            AttributeError: 'NoneType' object has no attribute 'attrib'
                                            -->End of Python script error report<--
10:38:11 T:140012753758080   ERROR: GetDirectory - Error getting plugin://plugin.audio.wfmu/archive/feed/KK/
10:38:11 T:140012753758080   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.audio.wfmu/archive/feed/KK/) failed
```

This is on OpenElec 6.0.98-fritch, Generic x86_64.

Thanks!

**Comment by sostenuti** [comment]
Sorry, I would have fixed this earlier, my GitHub notifications don't seem to have been working. It's fixed in 1.0.1

**Comment by github-ronk** [comment]
Thanks @sostenuti
